contracts/SSVNetwork.sol

```diff
@@ -354,7 +358,7 @@ contract SSVNetwork is UUPSUpgradeable, Ownable2StepUpgradeable, ISSVNetwork {
 
         --cluster.validatorCount;
 
-        delete validatorPKs[hashedValidator];
+        validatorPKs[hashedValidator].active = false;
```

Comment:
need to add
validatorPKs[hashedValidator].hashedOperatorIds = false

I think

Reply:
maybe this is better:
delete validatorPKs[hashedValidator]

Reply:
@lior-blox I suggested the delete and its updated to this, do we have a limitation in deleting the validator entry in removal?